client/state/ui/first-view/selectors.js

```diff
@@ -57,8 +53,7 @@ export function isViewEnabled( state, config ) {
 		return false;
 	}
 
-	const firstViewHistory = filter( getPreference( state, 'firstViewHistory' ), entry => entry.view === config.name );
-	const latestFirstViewHistory = [ ...firstViewHistory ].pop();
+	const latestFirstViewHistory = last( filter( getPreference( state, 'firstViewHistory' ), entry => entry.view === config.name ) );
```

Comment:
>findLast here too?

The only one I was hesitant to use `find` on. If we can determine whether `filter` would match a maximum of one, we could use `find`. Else, you're right, `findLast` is probably a better option.

cc @mjuhasz re: #6717